.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   unit-test:
     docker:
       - image: circleci/golang:1.13-buster
-      - image: circleci/postgres:11
```

Comment:
What version are we running in prod? I think it'd be good to match that.

Are we using any postgres 12 features?

Reply:
I _think_ we run 11 in prod with @Micheletto could confirm. I don't foresee using 12 features any time soon, so happy to roll it back if you think parity is more important than upgrading.

Reply:
I think it's better to test against the version we have deployed. We could test against postgres 11 and 12.